app/models/user.rb

```diff
@@ -1297,7 +1297,8 @@ def is_special_account?
       any_kind_of_delegate? ||
       !delegated_competitions.empty? ||
       !competitions_announced.empty? ||
-      !competitions_results_posted.empty?
+      !competitions_results_posted.empty? ||
+      board_member?
```

Comment:
Why is this change necessary? Feels like you're touching a method that should not be affected by your migration

Reply:
Actually board member is also considered as special account according to unit test. Earlier, this was covered in `self.teams.any`, but now since board is out of teams, this is necessary..

Reply:
Aha! Makes sense, thanks for the explanation.